mteb/tasks/Classification/multilingual/SIB200Classification.py

```diff
@@ -0,0 +1,263 @@
+    "ltg": ["ltg-Latn"],
+    "ltz": ["ltz-Latn"],
+    "lua": ["lua-Latn"],
+    "lug": ["lug-Latn"],
+    "luo": ["luo-Latn"],
+    "lus": ["lus-Latn"],
+    "lvs": ["lvs-Latn"],
+    "mag": ["mag-Deva"],
+    "mai": ["mai-Deva"],
+    "mal": ["mal-Mlym"],
+    "mar": ["mar-Deva"],
+    "min": ["min-Latn"],
+    "mkd": ["mkd-Cyrl"],
+    "mlt": ["mlt-Latn"],
+    "mni": ["mni-Beng"],
+    "mos": ["mos-Latn"],
+    "mri": ["mri-Latn"],
+    "mya": ["mya-Mymr"],
+    "nld": ["nld-Latn"],
+    "nno": ["nno-Latn"],
+    "nob": ["nob-Latn"],
+    "npi": ["npi-Deva"],
+    "nqo": ["nqo-Nkoo"],
+    "nso": ["nso-Latn"],
+    "nus": ["nus-Latn"],
+    "nya": ["nya-Latn"],
+    "oci": ["oci-Latn"],
+    "ory": ["ory-Orya"],
+    "pag": ["pag-Latn"],
+    "pan": ["pan-Guru"],
+    "pap": ["pap-Latn"],
+    "pbt": ["pbt-Arab"],
+    "pes": ["pes-Arab"],
+    "plt": ["plt-Latn"],
+    "pol": ["pol-Latn"],
+    "por": ["por-Latn"],
+    "prs": ["prs-Arab"],
+    "quy": ["quy-Latn"],
+    "ron": ["ron-Latn"],
+    "run": ["run-Latn"],
+    "rus": ["rus-Cyrl"],
+    "sag": ["sag-Latn"],
+    "san": ["san-Deva"],
+    "sat": ["sat-Olck"],
+    "scn": ["scn-Latn"],
+    "shn": ["shn-Mymr"],
+    "sin": ["sin-Sinh"],
+    "slk": ["slk-Latn"],
+    "slv": ["slv-Latn"],
+    "smo": ["smo-Latn"],
+    "sna": ["sna-Latn"],
+    "snd": ["snd-Arab"],
+    "som": ["som-Latn"],
+    "sot": ["sot-Latn"],
+    "spa": ["spa-Latn"],
+    "srd": ["srd-Latn"],
+    "srp": ["srp-Cyrl"],
+    "ssw": ["ssw-Latn"],
+    "sun": ["sun-Latn"],
+    "swe": ["swe-Latn"],
+    "swh": ["swh-Latn"],
+    "szl": ["szl-Latn"],
+    "tam": ["tam-Taml"],
+    "taq": ["taq-Tfng"],
+    "tat": ["tat-Cyrl"],
+    "tel": ["tel-Telu"],
+    "tgk": ["tgk-Cyrl"],
+    "tgl": ["tgl-Latn"],
+    "tha": ["tha-Thai"],
+    "tir": ["tir-Ethi"],
+    "tpi": ["tpi-Latn"],
+    "tsn": ["tsn-Latn"],
+    "tso": ["tso-Latn"],
+    "tuk": ["tuk-Latn"],
+    "tum": ["tum-Latn"],
+    "tur": ["tur-Latn"],
+    "twi": ["twi-Latn"],
+    "tzm": ["tzm-Tfng"],
+    "uig": ["uig-Arab"],
+    "ukr": ["ukr-Cyrl"],
+    "umb": ["umb-Latn"],
+    "urd": ["urd-Arab"],
+    "uzn": ["uzn-Latn"],
+    "vec": ["vec-Latn"],
+    "vie": ["vie-Latn"],
+    "war": ["war-Latn"],
+    "wol": ["wol-Latn"],
+    "xho": ["xho-Latn"],
+    "ydd": ["ydd-Hebr"],
+    "yor": ["yor-Latn"],
+    "yue": ["yue-Hant"],
+    "zho": ["zho-Hant"],
+    "zsm": ["zsm-Latn"],
+    "zul": ["zul-Latn"],
+}
+
+
+def _transform(dataset):
+    dataset = dataset.class_encode_column("category")
+    dataset = dataset.rename_columns({"category": "label"})
+    dataset = dataset.remove_columns(["index_id"])
+    print(len(dataset["test"]))
+    return dataset
+
+
+class SIB200Classification(MultilingualTask, AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="SIB200Classification",
+        description="""SIB-200 is the largest publicly available topic classification
+        dataset based on Flores-200 covering 205 languages and dialects.
```

Comment:
This sounds like a clustering dataset?

Reply:
I thought of it, but the `AbsTaskClustering`, requires "list of str", whereas here we have a single category assigned to each text. Hence I assumed it's N-class classification problem.